packages/graphql/src/resolvers/messaging/Messaging.js

```diff
@@ -57,24 +52,27 @@ export default {
   enabled: () => {
     return checkForMessagingOverride() ? messagingOverride.enabled : isEnabled()
   },
-  conversations: () => convosWithSupport(),
+  conversations: async (_, { limit, offset }) =>
+    await getConversationIds({ limit, offset }),
   conversation: (_, args) =>
     new Promise(async resolve => {
-      const convos = await getMyConvs()
-      if (!convos[args.id]) {
+      if (!(await contracts.messaging.conversationExists(args.id))) {
         resolve(null)
       }
-      resolve({ id: args.id, timestamp: Math.round(convos[args.id] / 1000) })
+
+      resolve({
+        id: args.id,
+        // timestamp: Math.round(convos[args.id] / 1000),
```

Comment:
is that intentionally commented out ?

Reply:
Yes, We use last message's timestamp as conversation timestamp. So this isn't necessary. Will remove that